node/node.go

```diff
@@ -179,6 +179,8 @@ func (n *Node) Start(ctx context.Context) error {
 		if err != nil {
 			return fmt.Errorf("failed to register the operator: %w", err)
 		}
+	} else {
+		n.Logger.Info("The node is started, but it's not opt-in in EigenDA yet (it's not receiving and validating data in EigenDA). To register, please follow the guide in https://docs.eigenlayer.xyz/operator-guides/avs-installation-and-registration/eigenda-operator-guide/eigenda-avs-installation-and-registration")
```

Comment:
Super helpful!

There might be a typo at the "opt-in in" vs "opt-in to"?

If we can avoid direct links to the docs URL that would be safest. Those URLs are unfortunately, subject to change in the near-mid term.

Reply:
Replaced it with the github repo which should be stable.